near-sdk-macros/src/lib.rs

```diff
@@ -213,6 +213,140 @@ pub fn metadata(item: TokenStream) -> TokenStream {
     }
 }
 
+#[proc_macro_derive(NearAbi, attributes(abi, serde, borsh_skip, schemars, validate))]
+pub fn derive_near_abi(_input: TokenStream) -> TokenStream {
+    // todo! borsh & serde don't support unions
+    // todo! guess what? we don't either
```

Comment:
Yeah, I meant, since neither `borsh` nor `serde` (via `schemars`) support unions, we need to make sure to return a neat error saying unions are not supported for the `NearSchema` derive macro too.